crates/ln-dlc-node/src/ln/config.rs

```diff
@@ -64,7 +67,7 @@ pub fn coordinator_config() -> UserConfig {
             // Enforces incoming channels to the coordinator to be public! We
             // only want to open private channels to our 10101 app.
             force_announced_channel_preference: true,
-            // lnd's max to_self_delay is 2016, so we want to be compatible.
+            // LND's max to_self_delay is 2016, so we want to be compatible.
```

Comment:
🔧 I think we also don't want to accept channels with `2016` blocks delay opened to us. It might be safe to reduce that value to less? but not exceed 2016. We could easily test that with our faucet setup.

Reply:
Yeah, we might want to play around with this number, although I didn't do so with this PR.